docs/instrumentation-list.yaml

```diff
@@ -3046,6 +3046,38 @@ libraries:
       description: Whether to instrument all classes that implement the Executor interface.
       type: boolean
       default: false
+  failsafe:
+  - name: failsafe-3.0
+    description: This standalone instrumentation enables metrics for Failsafe circuit
```

Comment:
I guess you used `standalone instrumentation` to emphasize that this is only a library instrumentation. Other library only instrumentations use different language.